app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -219,3 +227,45 @@ export default compose(
   withLocalState,
   withLocalMutation,
 )(Scene);
+
+// eslint-disable-next-line no-underscore-dangle
+Sleeping._motionSleepThreshold = 0.5;
+
+// eslint-disable-next-line func-names
+Sleeping.update = function(bodies, timeScale) {
```

Comment:
Not seeing the need for copying over this code -- doesn't look like anything changes except our linting rules?

If that's the case, I think `Sleeping._motionSleepThreshold = 0.5;` should be enough.

Reply:
Sorry, putting that function there was a bit unclear on my part, but it needs to be overwritten to enforce setting the bodies to static instead of putting them to sleep.
```suggestion
        if (body.sleepCounter >= body.sleepThreshold) {
          Body.setStatic(body, true);
        }
```
versus the original ... 
```suggestion
                if (body.sleepCounter >= body.sleepThreshold)
                    Sleeping.set(body, true);
```

Reply:
In that case, lets name the function for what it does differentlyand then assign it to `Sleeping.update` which would also address the func-names eslint violation.

Reply:
How does this sound?
`Sleeping.update = function freezeOnSleep(bodies, timeScale) {`

Reply:
maybe `staticOnSleep`? (unless freeze and static are used interchangeably in matterjs code/docs). Also, we're already patching `Sleeping.update` -- could we do:

```javascript
Sleeping._motionSleepThreshold = 0.5;
Sleeping.set = (param1, param2)=> Body.setStatic(body, true);
```
(that closure above^ is _probably_ necessary and I can go through it if it's not clear why)